tests/models/imagebind/test_processor_imagebind.py

```diff
@@ -228,3 +242,204 @@ def test_model_input_names(self):
+        feature_extractor = self.get_component("feature_extractor")
+
+        processor = self.processor_class(tokenizer=tokenizer, image_processor=image_processor, feature_extractor=feature_extractor)
+        self.skip_processor_without_typed_kwargs(processor)
+
+        input_str = "lower newer"
+        image_input = self.prepare_image_inputs()
+        inputs = processor(
+            text=input_str,
+            images=image_input,
+            return_tensors="pt",
+            crop_size={"height": 214, "width": 214},
+            padding="max_length",
+            max_length=76,
+        )
+
+        self.assertEqual(inputs["pixel_values"].shape[2], 214)
+        self.assertEqual(len(inputs["input_ids"][0]), 76)
+
+    @require_torch
+    @require_vision
+    def test_unstructured_kwargs_batched(self):
+        if "image_processor" not in self.processor_class.attributes:
+            self.skipTest(f"image_processor attribute not present in {self.processor_class}")
+        image_processor = self.get_component("image_processor")
+        tokenizer = self.get_component("tokenizer")
+        feature_extractor = self.get_component("feature_extractor")
+
+        processor = self.processor_class(tokenizer=tokenizer, image_processor=image_processor, feature_extractor=feature_extractor)
+        self.skip_processor_without_typed_kwargs(processor)
+
+        input_str = ["lower newer", "upper older longer string"]
+        image_input = self.prepare_image_inputs() * 2
+        inputs = processor(
+            text=input_str,
+            images=image_input,
+            return_tensors="pt",
+            crop_size={"height": 214, "width": 214},
+            padding="longest",
+            max_length=76,
+        )
+
+        self.assertEqual(inputs["pixel_values"].shape[2], 214)
+
+        self.assertEqual(len(inputs["input_ids"][0]), 6)
+
+    @require_torch
+    @require_vision
+    def test_doubly_passed_kwargs(self):
+        if "image_processor" not in self.processor_class.attributes:
+            self.skipTest(f"image_processor attribute not present in {self.processor_class}")
+        image_processor = self.get_component("image_processor")
+        tokenizer = self.get_component("tokenizer")
+        feature_extractor = self.get_component("feature_extractor")
+
+        processor = self.processor_class(tokenizer=tokenizer, image_processor=image_processor, feature_extractor=feature_extractor)
+        self.skip_processor_without_typed_kwargs(processor)
+
+        input_str = ["lower newer"]
+        image_input = self.prepare_image_inputs()
+        with self.assertRaises(ValueError):
+            _ = processor(
+                text=input_str,
+                images=image_input,
+                images_kwargs={"crop_size": {"height": 222, "width": 222}},
+                crop_size={"height": 214, "width": 214},
+            )
+
+    @require_torch
+    @require_vision
+    def test_structured_kwargs_nested(self):
```

Comment:
Are those tests overriden from `ProcessorTesterMixin`? If yes why do we need to override them?

Reply:
Yes those are overriden because in the original feature extractor has not been declared and pass to the processor_class:
https://github.com/huggingface/transformers/blob/74e19e81e2a23809af192532b9b0e7ea202be6f2/tests/test_processing_common.py#L149